scripts/leapfrog/pre_redeploy.sh

```diff
@@ -73,6 +87,11 @@ pushd ${LEAPFROG_DIR}
                 log "user_secrets_migration" "skipped"
             fi
         popd
+        if [[ "${IS_GATING}" != "TRUE" ]]; then
+            notice "Please verify your migrated secrets in /etc/openstack_deploy"
+            warning "DO NOT CTRL-C this process to verify your secrets."
```

Comment:
You're on your own then.
The idea is to never interrupt the leapfrog.

It should be fine theoretically, because the process will skip this part (this warning happened at the end) + the parts before.
Interrupting and restarting will continue to the next steps, and not show this warning anymore.

However, some files were already migrated, and some containers were not, so we are sitting between two chairs, and it can be weird.